packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -8,14 +8,20 @@ export interface ICheckbox {
   checked: boolean;
 
   /** Sets focus to the checkbox. */
-  focus(): void;
+  focus: () => void;
 }
 
 /**
  * Checkbox properties.
  */
 export interface ICheckboxProps extends React.HTMLProps<HTMLElement | HTMLInputElement> {
   /**
+   * Optional way to fetch the IButton interface. Use this instead of ref, to avoid accessing higher-order component
+   * wrappers rather than the IButton interface.
+   */
+  componentRef?: (component: ICheckbox) => void;
+
+  /**
```

Comment:
As discussed offline, I think it would be good if we could move this to an interface

```TypeScript
interface IComponentRef<TComponent> {
  componentRef?: (component: TComponent)  => void;
}
```

and then add it to the component's props by extending multiple interfaces? Or by `&` it in `BaseComponent`? Shouldn't be a blocker for this change, but going forward it would be nice to centralize these shared props to make refactoring etc. easier.